lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -324,8 +327,8 @@ def decrement_positions_on_higher_items(position)
 
         # This has the effect of moving all the lower items up one.
         def decrement_positions_on_lower_items(position=nil)
-          return unless in_list?
-          position ||= send(position_column).to_i
+          position ||= send(position_column)
```

Comment:
We're not converting the position into an integer anymore. Can you explain why? :)

Reply:
I can't think of a reason to drop that check; will reinstate it.